src/DataFetcher.class.js

```diff
@@ -35,6 +37,7 @@ export default class DataFetcher {
     this.lastUpdate = new Date(data.lastUpdate);
 
     if (this.lastUpdate.getTime() < Date.now() - (10 * 60 * 60 * 1000)) {
+      logger.error("Cache is outdated")
```

Comment:
```suggestion
      logger.error("Cache is outdated");
```
Take care to the linter.

Also, this log is not really useful, it doesn't give more informations compared to the error thrown. We can either skip it or improve it by giving more informations for instance: since when is the cache outdated ?